e2e/.eslintrc

```diff
@@ -6,7 +6,27 @@
     "no-console": 0,
     "@typescript-eslint/no-namespace": "off",
     "cypress/no-async-tests": "error",
-    "quotes": ["error", "double", { "avoidEscape": true }]
+    "quotes": ["error", "double", { "avoidEscape": true }],
+    "no-restricted-imports": [
+      "error",
+      {
+        "paths": [
```

Comment:
if we do not mention moment, it will be lost and not merged with the root config